app/interceptors/ExceptionInterceptor.java

```diff
@@ -72,7 +71,7 @@ private int getStatusCode(final Throwable throwable) {
     private String getMessage(final Throwable throwable, final int status) {
         String message = throwable.getMessage();
         if (StringUtils.isBlank(message)) {
-            message = HttpStatus.getStatusText(status);
+            message = Integer.toString(status);
```

Comment:
As far as I know, the exceptions need to store StatusLine (or the reason phrase (StatusLine.getReasonPhrase()) plus status code).  The change is probably not trivial.